updated_readme.md

```diff
@@ -0,0 +1,150 @@
+# Design Choices
```

Comment:
What's this file for? The name is a little odd.

Reply:
My bad, accidentally added unnecessary things again. Will delete soon.